core/platform/email/mailgun_email_services_test.py

```diff
@@ -53,3 +58,129 @@ def test_sending_email(self):
             mailgun_email_services.send_mail(
                 feconf.SYSTEM_EMAIL_ADDRESS, feconf.ADMIN_EMAIL_ADDRESS,
                 'subject', 'body', 'html', bcc_admin=False)
+
+
+    def test_sending_email_data(self):
+        """Verifies that the data sent in send_mail is correct."""
+        mailgun_api = self.swap(feconf, 'MAILGUN_API_KEY', 'api')
+        mailgun_domain = self.swap(feconf, 'MAILGUN_DOMAIN_NAME', 'domain')
+        allow_emailing = self.swap(feconf, 'CAN_SEND_EMAILS', True)
+
+        # This data should have been sent in the requests.post().
+        expected = {'from': feconf.SYSTEM_EMAIL_ADDRESS, 'to':
```

Comment:
can you format this to have one key-value pair per line? Easier to read that way.